assignment1/palindrome.rb

```diff
@@ -0,0 +1,16 @@
+def palindrome sentence
+    sentence_without_space = sentence.delete(' ').downcase
+    palindrome = '';
+    sent_length = sentence_without_space.length
+
+    (sent_length).times { |i|
+        palindrome += sentence_without_space[-i - 1].to_s
+    }
+
+    puts palindrome
+    puts sentence_without_space
+    puts palindrome == sentence_without_space ? 'true' : 'false'
+
+end
+
```

Comment:
this also should be returning boolean value true or false instead of string "true" or "false"